src/modules/createGames.js

```diff
@@ -0,0 +1,16 @@
+const createGame = async (url, name, score) => {
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({
+      user: name,
+      score,
+    }),
+  })
+    .then((response) => response.json())
+    .then((json) => json);
+};
+
```

Comment:
[OPTIONAL] Well done overall 👍🏽 but look to add a `catch` clause here to manage the error handling part. And maybe show a message in case of an error. 😉